docs/docsite/rst/scenario_guides/guide_gce.rst

```diff
@@ -3,99 +3,106 @@ Google Cloud Platform Guide
 
 .. gce_intro:
 
-Introduction
-------------
+Auto-generated GCP Modules
+--------------------------
+
+Ansible + Google have been working together on a set of auto-generated
+Ansible modules designed to consistently and comprehensively cover the entirety
+of the Google Cloud Platform.
+
+Ansible contains modules for managing Google Cloud Platform resources,
+including creating instances, controlling network access, working with
+persistent disks, managing load balancers, and a lot more.
+
+These new modules can be found under a new consistent name scheme "gcp_*"
+(Note: gcp_target_proxy and gcp_url_map are legacy modules, despite the "gcp_*"
+name. Please use gcp_compute_target_proxy and gcp_compute_url_map instead).
 
-.. note:: This section of the documentation is under construction. We are in the process of adding more examples about all of the GCE modules and how they work together. Upgrades via github pull requests are welcomed!
+Additionally, the gcp_compute inventory plugin can discovery all GCE instances
+and make them automatically available in your Ansible inventory.
 
-Ansible contains modules for managing Google Compute Engine resources, including creating instances, controlling network access, working with persistent disks, and managing
-load balancers.  Additionally, there is an inventory plugin that can automatically suck down all of your GCE instances into Ansible dynamic inventory, and create groups by tag and other properties.
+You may see a collection of other GCP modules that do not conform to this
+naming convention. These are the original modules primarily developed by the
+Ansible community. You will find some overlapping functionality such as with
+the the "gce" module and the new "gcp_compute_instance" module. Either can be
+used, but you may experience issues trying to use them together.
```

Comment:
It's more of a hypothetical that they won't work well together.

Both sets of modules do the same things: altering things in the cloud using the GCP APIs. Because they call the same APIs, I see no reason why you couldn't use the different modules at the same time. It just wouldn't be as pleasant of an experience.